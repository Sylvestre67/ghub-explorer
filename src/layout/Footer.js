import React from 'react';
import PropTypes from 'prop-types';

import { withStyles } from '@material-ui/core';

import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

const isEven = process.env.REACT_APP_IS_EVEN;

const styles = theme => ({
	footer: {
		backgroundColor: theme.palette.primary.main,
		color: 'white',
		'& > div': {
			justifyContent: 'center',
		},
	},
});

export function Footer(props) {
	const { classes } = props;

	return (
		<footer className={classes.footer}>
			<Toolbar elevation={0}>
				<Typography variant="caption" color="inherit">
					{isEven
						? 'â’¸ 2017 Even Financial, Inc. - CONFIDENTIAL'
						: 'Made With Love in NYC'}
				</Typography>
			</Toolbar>
		</footer>
	);
}

Footer.propTypes = {
	classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(Footer);
