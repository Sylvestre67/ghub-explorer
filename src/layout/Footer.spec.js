import React from 'react';
import { shallow } from 'enzyme';

import { Footer } from './Footer';
let component;

const props = {
	classes: {
		grow: '',
	},
};

describe('Footer', () => {
	beforeAll(() => {
		component = shallow(<Footer {...props} />);
	});

	it('should render correctly', () => {
		expect(component).toMatchSnapshot();
	});
});
