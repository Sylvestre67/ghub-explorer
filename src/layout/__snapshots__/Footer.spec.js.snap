// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer>
  <WithStyles(Toolbar)
    elevation={0}
  >
    <WithStyles(Typography)
      color="inherit"
      variant="caption"
    >
      Made with love in NYC
    </WithStyles(Typography)>
  </WithStyles(Toolbar)>
</footer>
`;
